### PR TITLE
Update seat-management.mdx

### PR DESCRIPTION
**Pull request by kennyj42**
This was incorrect, we look at the last seen value from gateway

### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

File: src/content/docs/cloudflare-one/identity/users/seat-management.mdx [hunk 1]
```diff
@@ -11,7 +11,7 @@ The amount of user seats available in your Zero Trust account depends on the amo
 
 ## What constitutes an authentication event
 
-For Access, this is any Cloudflare Access authentication event, like a login to the [App Launcher](/cloudflare-one/applications/app-launcher/) or an application. For Gateway, this means any Cloudflare WARP authentication event, like enrolling a device to your ZT organization.
+For Access, this is any Cloudflare Access authentication event, like a login to the [App Launcher](/cloudflare-one/applications/app-launcher/) or an application. For Gateway, this is if any devices associated have been seen in the specified period.
```
Comment (jroyal) [inline src/content/docs/cloudflare-one/identity/users/seat-management.mdx]:
```suggestion
For Access, this is any Cloudflare Access authentication event, like a login to the [App Launcher](/cloudflare-one/applications/app-launcher/) or an application. For Gateway, this is if any devices associated with the user have been seen in the specified period.
```

Comment (ranbel) [reply src/content/docs/cloudflare-one/identity/users/seat-management.mdx]: What is the `specified period`?

**Comment by ranbel** [comment]
relates to #18081

File: src/content/docs/cloudflare-one/identity/users/seat-management.mdx (hunk 1 above)
Comment (jroyal) [reply src/content/docs/cloudflare-one/identity/users/seat-management.mdx]: The period is the setting in https://developers.cloudflare.com/cloudflare-one/identity/users/seat-management/#enable-seat-expiration.

However after looking at this page again I think we should leave the original statement. That _is_ what constitutes an authentication event. However we consider any seat active for the purposes of seat expiration if they have any devices associated that have been seen. I'm not sure how we should rework the page for that.

Comment (ranbel) [reply src/content/docs/cloudflare-one/identity/users/seat-management.mdx]: > However we consider any seat active for the purposes of seat expiration if they have any devices associated that have been seen.

@maxvp perhaps this info could go into the Seat Expiration section:
https://github.com/cloudflare/cloudflare-docs/pull/18081/files#diff-50a1ac1c6b0bd5ec93cc24a705dc80e9bed38d044bc889dc39d41eba6bdb2ce1R60

**Comment by maxvp** [comment]
@kennyj42 @jroyal I recently updated this page via #18081. Do the changes in this PR still apply?

**Comment by kennyj42** [comment]
Yes they would. Your change still has the "or trigger a WARP enrollment" portion which isn't correct.

**Review by maxvp — APPROVED** [review]

**Pull request merged**